### ISSUE TITLE
Comment option

### ISSUE THREAD

**Issue by kevva** [thejoshwolfe/yazl#15]
Might be useful for some people. Something like JSZip has https://stuk.github.io/jszip/documentation/api_jszip/file_data.html.

**Comment by thejoshwolfe** [comment]
would it be useful to you? can you give me an example usecase for the comment field being helpful?

**Comment by kevva** [comment]
To me personally, no, but we recently switched to your library in https://github.com/sindresorhus/gulp-zip and it was requested there once. Not a super big deal, but a nice to have thing.

**Comment by thejoshwolfe** [comment]
It looks like nobody knows why this feature would be helpful. I'll certainly add it if there's a compelling usecase, but I won't add the feature just to compete with another library's list of features. Adding a useless feature (assuming the feature is useless) clutters up the code, api docs, and tests, and has a slight chance of causing complications with various unzip implementations, which are quite difficult to detect and debug.

In summary, I'm happy to add the feature if it will be valuable, bot not otherwise.

**Comment by kevva** [comment]
Can't argue with you. I agree it's just glossary. I'll ping you again if we get any compelling feature requests that this really need to be added.
